tests/network/vmi_multus.go

```diff
@@ -332,26 +353,55 @@ var _ = SIGDescribe("[Serial]Multus", func() {
 		})
 
 		Context("VirtualMachineInstance with Linux bridge plugin interface", func() {
-			table.DescribeTable("should be able to ping between two vms", func(interfaces []v1.Interface, networks []v1.Network, ifaceName string) {
+			getInterfaceAddressByNetworkName := func(vmi *v1.VirtualMachineInstance, networkName string) (string, error) {
```

Comment:
Can't you just pass the vmiName as a parameter instead of the whole VMI ? AFAIU, you're not using any other parameter.

I also miss the word `IP` in the name of this function. `getIfaceIPByNetworkName` maybe ?